upstream-community-operators/infinispan/infinispan-operator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,181 @@
+#! validate-crd: deploy/chart/templates/0000_30_02-clusterserviceversion.crd.yaml
+#! parse-kind: ClusterServiceVersion
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: infinispan-operator.v0.1.0
+  namespace: placeholder
+  annotations:
+    tectonic-visibility: ocs
+    capabilities: Basic Install
+    categories: "Database"
+    alm-examples: |
+      [
+        {
+          "apiVersion": "infinispan.org/v1",
+          "kind": "Infinispan",
+          "metadata": {
+            "name": "example-infinispan"
+          },
+          "spec": {
+            "size": 3
+          }
+        },
+        {
+          "apiVersion": "infinispan.org/v1",
+          "kind": "Infinispan",
+          "metadata": {
+            "name": "example-infinispan"
+          },
+          "config": {
+            "sourceType": "ConfigMap",
+            "sourceRef": "infinispan-app-configuration",
+            "name": "cloud-ephemeral.xml"
+          },
+          "spec": {
+            "size": 3
+          }
+        }
+      ]
+    description: TBD
```

Comment:
Leaving a comment here since if this is merged, this would be displayed in the operator tile in operatorhub.io

Reply:
This is not final submission, @oraNod and I will complete all `TBD` sections.